frontend/src/app/playback/playback.component.css

```diff
@@ -1,10 +1,13 @@
 /* src/app/playback/playback.component.css */
 
-.container {
+--first-color: #fcf9f4;
+--third-color: #ea7659;
+
+:host {
```

Comment:
You can style the component itself by targeting `:host` and lose the top level 'container' divs in your components